include/oneapi/tbb/task_arena.h

```diff
@@ -404,6 +498,33 @@ class task_arena : public task_arena_base {
         return execute_impl<decltype(f())>(f);
     }
 
+#if __TBB_PREVIEW_PARALLEL_PHASE
+    void start_parallel_phase() {
+        initialize();
+        r1::register_parallel_phase(this, /*reserved*/0);
+        // Trigger worker threads to join arena
+        enqueue([]{});
+    }
+    void end_parallel_phase(bool with_fast_leave = false) {
+        __TBB_ASSERT(my_initialization_state.load(std::memory_order_relaxed) == do_once_state::initialized, nullptr);
+        r1::unregister_parallel_phase(this, with_fast_leave);
```

Comment:
I would make the conversion to uintptr_t explicit (even though the implicit one works) and even would add a comment that the passed value is either 0 or 1 (it's guaranteed by the standard, but does everybody remember it?)

Reply:
Done.